e2e/__tests__/jest_changed_files.test.js

```diff
@@ -326,3 +347,31 @@ test('gets changed files for hg', async () => {
       .sort(),
   ).toEqual(['file5.txt']);
 });
+
+test('should only monitor root paths for hg', async () => {
```

Comment:
we should avoid using `should` in test titles :)
use `it('monitors only root paths for hg'...`